From: =?utf-8?q?Timo_R=C3=B6hling?= <roehling@debian.org>
Date: Thu, 22 Feb 2024 13:29:06 +0100
Subject: Fudge rust crate version requirements

Forwarded: not-needed
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 661240c..76ffb72 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -45,7 +45,7 @@ idna = "1.0.2"
 base64 = "0.22.1"
 num-bigint = "0.4.6"
 python3-dll-a = "0.2.10"
-uuid = "1.11.0"
+uuid = "1.10.0"
 jiter = { version = "0.7.1", features = ["python"] }
 hex = "0.4.3"
 
